components/brave_extension/extension/brave_extension/braveShieldsPanel.tsx

```diff
@@ -4,34 +4,39 @@
 
 import * as React from 'react'
 import * as ReactDOM from 'react-dom'
-
-import Theme from 'brave-ui/theme/brave-default'
-import { ThemeProvider } from 'brave-ui/theme'
-
+import shieldsDarkTheme from 'brave-ui/theme/shields-dark'
+import shieldsLightTheme from 'brave-ui/theme/shields-light'
 import { Provider } from 'react-redux'
 import { Store } from 'react-chrome-redux'
+import BraveCoreThemeProvider from '../../../common/BraveCoreThemeProvider'
 import BraveShields from './containers/braveShields'
 require('../../../fonts/muli.css')
 require('../../../fonts/poppins.css')
 
-chrome.storage.local.get('state', (obj) => {
```

Comment:
@cezaraugusto This didn't get used, so took the opportunity to remove. LMK if I'm missing something here...?